src/App/Router.php

```diff
@@ -202,6 +202,7 @@ public function collectRoutes()
 		$this->routeCollector->addRoute(['GET'],         '/smilies[/json]',      Module\Smilies::class);
 		$this->routeCollector->addRoute(['GET'],         '/statistics.json',     Module\Statistics::class);
 		$this->routeCollector->addRoute(['GET'],         '/tos',                 Module\Tos::class);
+		$this->routeCollector->addRoute(['GET'],         '/view/theme/{theme:.+}',  Module\Theme::class);
```

Comment:
This endpoint is used to load `view/theme/{theme}/style.pcss` URLs. This pattern would probably catch `{theme}/style.pcss` in the `theme` parameter. Also the `.+` is redundant with the default parameter catch pattern without the regular expression.

Reply:
I thought I have to make `style.pcss` more dynamic, so `style.css` would work too. But we do have just the first type, so I changed it to static `{theme}/style.pcss` :-)